packages/compass-explain-plan/src/components/explain-tree/explain-tree.tsx

```diff
@@ -59,10 +59,11 @@ const ExplainTree: React.FunctionComponent<ExplainTreeProps> = ({
       data={root}
       getNodeSize={getNodeSize}
       getNodeKey={getNodeKey}
-      linkColor={darkMode ? palette.gray.base : palette.gray.light2}
+      linkColor={darkMode ? palette.gray.dark2 : palette.gray.light2}
+      arrowColor={darkMode ? palette.gray.base : palette.gray.light1}
       linkWidth={6}
       horizontalSpacing={spacing[6]}
-      verticalSpacing={spacing[6]}
+      verticalSpacing={spacing[5] + 8}
```

Comment:
In the designs vertical spacing is variable: sharded name card is closer to child than other cards, here it's static. The `spacing` option of the `flextree` package takes function as an argument, not only number, we probably should use that to our advantage